fuse/eval/metrics/libs/stat.py

```diff
@@ -20,12 +20,22 @@ def pearson_correlation(
         :param target: target values
         :param mask: optional boolean mask. if it is provided, the metric will be applied only to the masked samples
         """
+        assert len(pred) > 0
         if isinstance(pred, Sequence):
-            pred = np.array(pred)
+            if np.isscalar(pred[0]):
+                pred = np.array(pred)
+            else:
+                pred = np.concatenate(pred)
```

Comment:
here pred is numpy? necessarily same dims?

Reply:
yes, in the else it's numpy, and it not necessarily same dims (it's 1d each)